pages-css/home-page-css/page-layout-handeler.css

```diff
@@ -0,0 +1,139 @@
+@import url(/pages-css/compo-css/navbar.css);
+@import url(/pages-css/home-page-css/home-page-style.css);
+:root {
+    --theam-background: rgb(206, 206, 206);
+    --theam-color: rgb(252, 1, 1);
+    --theam-border-radius: 15px;
+    --color-white: white;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Orienta', sans-serif;
+    font-family: 'Quicksand', sans-serif;
+    scroll-behavior: smooth;
+}
+
+
+/* component page container */
+
+.page-layout-container {
+    display: grid;
+    gap: 0.313rem;
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: 0.313rem;
+    row-gap: 0.313rem;
+    grid-template-areas: "header header header " "section-1 section-1 section-1 " "section-2 section-2 section-2 " "section-3 section-3 section-3 " "footer footer footer ";
```

Comment:
these would be more understandable if put in new lines for each area

Reply:
Ok